tests/integration/targets/nxos_prefix_lists/tests/common/_remove_config.yaml

```diff
@@ -1,10 +1,11 @@
 - name: Remove existing prefix-lists
   cisco.nxos.nxos_config:
-    lines:
-      - "no ip prefix-list AllowPrefix"
-      - "no ip prefix-list DenyPrefix"
-      - "no ip prefix-list AllowPrefix2Stub"
-      - "no ipv6 prefix-list AllowIPv6Prefix"
+    lines: "{{ item }}"
+  loop:
+    - "no ip prefix-list AllowPrefix"
```

Comment:
Making it a loop because otherwise if one line causes an error (because of the prefil-list not existing), the others aren't executed.